src/test/java/edu/cornell/kfs/module/purap/batch/service/impl/JaggaerXMLInputFileTypeTest.java

```diff
@@ -0,0 +1,63 @@
+package edu.cornell.kfs.module.purap.batch.service.impl;
+
+import edu.cornell.kfs.module.purap.batch.JaggaerXMLInputFileType;
+import edu.cornell.kfs.sys.util.LoadSpringFile;
+import edu.cornell.kfs.sys.util.SpringEnabledMicroTestBase;
+import org.junit.Test;
+import org.junit.jupiter.api.BeforeEach;
+
+import java.io.File;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+@LoadSpringFile("edu/cornell/kfs/module/purap/jaggaer/xml/cu-spring-jaggaer-test.xml")
+public class JaggaerXMLInputFileTypeTest extends SpringEnabledMicroTestBase {
```

Comment:
Ideally, it would be best to consolidate this test class with the `JaggaerUploadFileServiceImplTest` class, especially since all or most of this class's assertions could be consolidated into 1 test case. The Spring-loading feature could also be used to initialize the Jaggaer file type instance of the other test class.

If you wanted to try combining the test classes, you would either need to create a JUnit 5 version of the Spring-loading feature, or create a subclass of `SpringEnabledMicroTestBase` that can act as a pseudo-test-extension instance but has fully custom control of its initialization and configuration. The former would be preferable if it's not too complex.

Reply:
I see that the two test classes are still separate at the moment. I believe that's fine for now, given the separate test superclasses that are involved.

In the future, we should replace the `CuLocalServerTestBase` and `SpringEnabledMicroTestBase` superclasses with JUnit 5 extensions. That should simplify our unit test setup and also make it easier to consolidate these Jaggaer unit test classes. I can create a JIRA for further analysis.